### ISSUE TITLE
Please, add a slider 🙏

### ISSUE THREAD

**Issue by SupertigerDev** [jonls/redshift#856]
All i want is a simple slider in the redshift-gtk to control my temperature level easily. Please. I don't want it to be location based, I want to control it myself using a slider.

**Comment by munkiwarra** [comment]
I use a shellscript which I is invoked by shortcuts to lower and raise the color temperature and brightness  via
`redshift -P -O COLOR -B BRIGHTNESS`
This effects the current redshift instance immediatly without flickering.

I also would like to use a slider, although I think this would make more sense for the redshift-gtk application which already has a tray icon, as redshift has no gui.

**Comment by yllekz** [comment]
Yes please. I understand the purity of using config files and commands, but a GUI is just so much nicer for casual use and wider adoption.

**Comment by munkiwarra** [comment]
@yllekz 

Which distribution are you using?

There are different packages for arch which include partial  redshift configuration via GUI (see https://wiki.archlinux.org/title/redshift#Front_ends). Some are packed for other distributions as well, even though they might not be up to date (for example https://github.com/simgunz/redshift-plasmoid/).

**Comment by SupertigerDev** [comment]
Linux Mint

**Comment by yllekz** [comment]
[Linux Mint. ](https://linuxmint.com/)
